### PR TITLE
Ready for owncloud 10.0+

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -11,20 +10,15 @@
...
-    <bugs>https://github.com/zorn-v/nextcloud-social-login/issues</bugs>
-    <repository>https://github.com/zorn-v/nextcloud-social-login</repository>
-    <screenshot>https://raw.githubusercontent.com/zorn-v/nextcloud-social-login/master/appinfo/screenshot.png</screenshot>
+    <website>https://github.com/owncloud/sociallogin</website>
+    <bugs>https://github.com/owncloud/sociallogin/issues</bugs>
+    <repository>https://github.com/owncloud/sociallogin</repository>
+    <screenshot>https://raw.githubusercontent.com/owncloud/sociallogin/master/appinfo/screenshot.png</screenshot>
     <dependencies>
-        <nextcloud min-version="12" max-version="14" />
+        <owncloud min-version="10.1" max-version="11.0.0.0" />
```
Comment (tboerger) [inline appinfo/info.xml]: Only 10.1?

**Review by tboerger — COMMENTED** [review]
Your indentation looks wrong on github.

**Comment by DeepDiver1975** [comment]
> Your indentation looks wrong on github.

original code uses spaces - I use tabs - need to run php-cs-fixer ....

**Review by PVince81 — APPROVED** [review]
ok for now.

later on we'll set max-version to "10"

**Pull request merged**